frontend/src/router.js

```diff
@@ -18,6 +19,14 @@ export default new Router({
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+    },
+    {
+      path: '/question/:id',
+      name: 'question',
+      props: ({params: { id }}) => ({
```

Comment:
So we need to pass the id as a prop to this route, right? What does this function do and why?